### PR TITLE
Remove the entire archiving mechanism

### PR DESCRIPTION
**Pull request by ocr**
Delete all tests that are older than the configured parameter for archiving
tests

### CODE REVIEW

File: www/cli/archive.php [hunk 1]
```diff
@@ -224,26 +224,7 @@ function CheckTest($testPath, $id, $elapsedDays) {
   $elapsed = TestLastAccessed($id);
   if (isset($elapsed)) {
     if( $elapsed >= $MIN_DAYS ) {
-      if (ArchiveTest($id) ) {
```
Comment (renajohn) [inline www/cli/archive.php:227]: Shouldn't we delete function `ArchiveTest`?

File: www/cli/archive.php [hunk 2]
```diff
@@ -224,26 +224,7 @@ function CheckTest($testPath, $id, $elapsedDays) {
...
-        $archiveCount++;
-        $logLine .= "Archived";
-                                                                                      
-        if (VerifyArchive($id) || $elapsed >= 30)
```
Comment (renajohn) [inline www/cli/archive.php:231]: Should we delete `VerifyArchive` function?

**Comment by renajohn** [comment]
It seems fine to me. We could do some clean up by deleting dead code.

**Comment by ocr** [comment]
I didn't bother given the current plans for the codebase

**Comment by renajohn** [comment]
Fair enough

**Pull request merged**